.gitignore

```diff
@@ -462,4 +462,9 @@ dist/
 
 # Peristant storage
 data/qdrant
-data/chatstore*
\ No newline at end of file
+data/chatstore*
+
+# Java build
+java/**/target
+samples/java/**/target
+java/.mvn/wrapper/maven-wrapper.jar
```

Comment:
This is a potential source of merge conflicts when we automate merges from main. Can you 

1. Create a separate PR for this and we'll get this merged to main
2. Update the PR branch and merge it

I think this could save us to headaches as the experimental branch evolves.